### DISCUSSION TITLE
Adjusting Browser Settings and Extensions

### DISCUSSION THREAD

**Discussion by simeononsecurity** [simeononsecurity/Windows-Optimize-Debloat#8] [Q&A] [answered]
For @mrbateuk

https://github.com/simeononsecurity/Windows-Optimize-Debloat#editing-policies-in-local-group-policy-after-the-fact

### Editing policies in Local Group Policy after the fact:

If you need to modify or change a setting, they are most likely configurable via GPO:

- Import the ADMX Policy definitions from this [repo](https://github.com/simeononsecurity/STIG-Compliant-Domain-Prep/tree/master/Files/PolicyDefinitions) into _C:\windows\PolicyDefinitions_ on the system you're trying to modify.

- Open `gpedit.msc` on on the system you're trying to modify.

**Comment by simeononsecurity** [comment, marked as answer]
**Google Chrome:**
To configure Chrome settings using Group Policy, you will typically use Administrative Templates. The path for configuring Chrome settings is as follows:

1. Open the Group Policy Management Console (GPMC) on a Windows server or computer with the Group Policy Management feature installed.
2. Navigate to the following path:
   `Computer Configuration` > `Administrative Templates` > `Google` > `Google Chrome`

Here, you will find various settings related to Chrome.

**Microsoft Edge (Chromium):**
Configuring Microsoft Edge (Chromium) settings using GPO is similar to Chrome because the Chromium-based Edge browser uses the same engine as Chrome. The path is similar to the one for Chrome:

1. Open the Group Policy Management Console.
2. Navigate to the following path:
   `Computer Configuration` > `Administrative Templates` > `Microsoft Edge`

Here, you can find policies specific to the Chromium-based Microsoft Edge browser.

**Firefox:**
Configuring Firefox using GPO is a bit different. You would typically use a custom .adm or .admx file provided by Mozilla to manage Firefox settings through Group Policy. Mozilla provides a pre-built .admx file that you can use to create and deploy GPOs for Firefox.

1. Download the .admx files and corresponding .adml files from the Mozilla website.
2. Copy the .admx file to the "C:\Windows\PolicyDefinitions" directory on your Group Policy management machine.
3. Copy the .adml file to the "C:\Windows\PolicyDefinitions\en-US" directory (adjust the language folder as needed).

After this, you can configure Firefox settings by navigating through Group Policy:

`Computer Configuration` > `Administrative Templates` > `Mozilla`

In the "Mozilla" section, you will find the settings you can configure for Firefox.

Please note that the specific settings and paths may vary depending on the versions of these browsers and the version of your Group Policy management software. Make sure to use the most up-to-date GPO templates for these browsers and regularly check for updates from the browser vendors for any changes in settings or paths.

**Reply by mrbateuk** [reply]
Hi

Thank you,

However, there is no Google Chrome or Microsoft Edge underneath 
Computer Configuration > Administrative Templates 

<a href="https://imgbb.com/"><img src="https://i.ibb.co/jZT1gdP/1.jpg" alt="1" border="0"></a>

**Reply by simeononsecurity** [reply]
Because you didn't read and complete the steps.
Again, slow down. Read. Follow instructions.
https://github.com/simeononsecurity/Windows-Optimize-Debloat/discussions/8#discussion-5738764
